pkg/limayaml/default.yaml

```diff
@@ -27,11 +27,15 @@ images:
 - location: "https://cloud-images.ubuntu.com/impish/current/impish-server-cloudimg-arm64.img"
   arch: "aarch64"
 
-# Specify QEMU CPU emulation, e.g., "host", "qemu64", "Haswell".
+# Specify QEMU CPU type for each arch. Only used for emulation; cpu type is always "host"
+# when the desired arch matches the host arch.
```

Comment:
We have to support overriding "host" to "Haswell-v4":

https://github.com/lima-vm/lima/blob/71861bcc2c05c88c9cc2e74793df5354729c57d3/examples/rocky.yaml#L3-L5

Reply:
Thanks! Should be fixed now.